akka-docs/src/main/paradox/stream/operators/StreamConverters/fromJavaStream.md

```diff
@@ -1,17 +1,20 @@
 # StreamConverters.fromJavaStream
 
-Create a source that wraps a Java 8 `Stream`.
+Create a source that wraps a Java 8 `java.util.stream.Stream`.
 
 @ref[Additional Sink and Source converters](../index.md#additional-sink-and-source-converters)
 
-@@@div { .group-scala }
-
 ## Signature
 
-@@signature [Source.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/StreamConverters.scala) { #fromJavaStream }
+@apidoc[StreamConverters](StreamConverters$) { scala="#fromJavaStream[T,S%3C:java.util.stream.BaseStream[T,S]](stream:()=%3Ejava.util.stream.BaseStream[T,S]):akka.stream.scaladsl.Source[T,akka.NotUsed]" java="#fromJavaStream(akka.japi.function.Creator)" }
```

Comment:
`%3C` in the `scala` version of the link is wrong but `<` is not supported either.

See https://github.com/lightbend/sbt-paradox-apidoc/pull/130